src/vaadin-chart.html

```diff
@@ -1035,7 +1048,7 @@
         /**
          * Iterates over axes (y or x) and removes whenever it doesn't contain any series and was created for unit
          *
-         * @param {Boolean}  isXAxis indicates if it will remove x or y axes. Defaults to `false`.
+         * @param {boolean}  isXAxis indicates if it will remove x or y axes. Defaults to `false`.
          */
         __removeAxisIfEmpty(isXAxis) {
```

Comment:
`__removeAxisIfEmpty()` is called from vaadin-chart-series so should be `@protected`?

Reply:
Done.